### PR TITLE
Update dev packages of metacoq

### PR DESCRIPTION
**Pull request by mattam82**

### CODE REVIEW

**Comment by mattam82** [comment]
Needs metacoq/metacoq#303 first

File: extra-dev/packages/coq-metacoq-checker/coq-metacoq-checker.8.8.dev/opam [hunk 1]
```diff
@@ -36,6 +37,7 @@ relation along with a reference type-checker that is extracted to a pluging.
 This provides a command: `MetaCoq Check [global_reference]` that can be used
 to typecheck a Coq definition using the verified type-checker.
 """
-url {
-   src: "git+https://github.com/MetaCoq/metacoq.git#coq-8.8"
-}
+# url {
+#   src: "https://github.com/MetaCoq/metacoq/archive/v2.1-beta3.tar.gz"
+#   checksum: "md5=e81b8ecabef788a10337a39b095d54f3"
+# }
```
Comment (gares) [inline extra-dev/packages/coq-metacoq-checker/coq-metacoq-checker.8.8.dev/opam]: How is this supposed to work?

Comment (mattam82) [reply extra-dev/packages/coq-metacoq-checker/coq-metacoq-checker.8.8.dev/opam]: Sorry, I should have made a draft PR, this is not ready

**Comment by mattam82** [comment]
Ok, should be ready for review once it passes CI

**Comment by gares** [comment]
one job timed out because of a network problem, I've restarted it.

**Comment by mattam82** [comment]
Ah, it's cool that one pipeline reached the end. However, this is highly inefficient because we're testing all package installs rather than just the `coq-metacoq` metapackage one. In principle partial configs of packages could fail if we don't have the right deps, so I don't know how to solve this nicely.

**Pull request merged**